.eslintrc.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  env: {
+    browser: true,
+    es2021: true,
+    node: true,
+    es6: true,
+    'jest/globals': true,
+  },
+  extends: ['airbnb-base', 'plugin:jest/recommended'],
+  parserOptions: {
+    ecmaVersion: 13,
+  },
+  rules: {
+    'max-len': 0,
+    'no-unused-vars': 'warn',
+    quotes: ['error', 'double', { avoidEscape: true }],
+    'no-console': 'off',
+  },
+  plugins: ['jest', 'prettier'],
```

Comment:
Looks like you are now using prettier - I would try to stick with eslint only for this project (disable prettier for the vs code workspace). eslint and prettier can conflict with each other.